pages/en/lb4/Extending-LoopBack-4.md

```diff
@@ -0,0 +1,110 @@
+---
+lang: en
+title: 'Extending LoopBack 4'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Extending-LoopBack-4.html
+summary:
+---
+
+## Overview
+
+LoopBack 4 is designed to be highly extensible. For architectural rationale and motivation, see [Crafting LoopBack 4](Crafting-LoopBack-4.html).
+
+## Building blocks for extensibility
+
+The `@loopback/context` module implements an IoC container (`Context`) as a service registry that supports [Dependency injection](Dependency-injection.html).
```

Comment:
I assume "IoC" means inversion of control. If so, please spell out on first use and link to definition:

```
...implements an [inversion of control]([IoC](https://en.wikipedia.org/wiki/Inversion_of_control)) (IoC) container ...